trackbasket_be/models/krogerservice.py

```diff
@@ -69,16 +69,14 @@ def item_search(cls, term, at_risk_user_id):
       data['name'] = item['description']
       data['productId'] = item['productId']
       data['upc'] = item['upc']
-      
-      for price in item['items']:
-        data['unit_price'] = price['price']['regular']
-        
-      for aisle in item['aisleLocations']:
-        data['aisle_number'] = int(aisle['number']) 
-        
+      data['aisle_number'] = int(item['aisleLocations'][0]['number']) 
+      data['unit_price'] = item['items'][0]['price']['regular']
       for image in item['images']:
-        data['image'] = image['sizes'][0]['url']    
-      
+        if image['perspective'] == 'front': 
+          for size in image['sizes']: 
+            if size['size'] == 'medium':
+              data['image'] = size['url']
```

Comment:
specify value needed from hash to return the correct url

Reply:
I think we might be able to convert the part of the response to a string and split that string to extract the information we need without having a nested loop